File: src/routes/ateneaExecution.js
const routering = require('express').Router();
const currentHostRunsExecutions = require('../controllers/ateneaExecution')

routering.get('/prevMonht/ateneaExecution',(req,res)=>{
    currentHostRunsExecutions.listarPrevMonht(req,res);
})

routering.get('/lastMonht/ateneaExecution',(req,res)=>{
    currentHostRunsExecutions.listarLastMonht(req,res);
})

module.exports = routering

File: src/routes/ateneaExecution.py
from flask import Blueprint, request, jsonify
from controllers.ateneaExecution import listarPrevMonth, listarLastMonth

routerAteneaExecution = Blueprint('routering', __name__)

@routerAteneaExecution.route('/prevMonht/ateneaExecution', methods=['GET'])
def prevMonthAteneaExecutionRoute():
    return listarPrevMonth(request, jsonify())

@routerAteneaExecution.route('/lastMonht/ateneaExecution', methods=['GET'])
def lastMonthAteneaExecutionRoute():
    return listarLastMonth(request, jsonify())